amd_openvx/openvx/include/vx_ext_amd.h

```diff
@@ -146,6 +146,8 @@ enum vx_graph_attribute_amd_e {
     VX_GRAPH_ATTRIBUTE_AMD_PERFORMANCE_INTERNAL_PROFILE = VX_ATTRIBUTE_BASE(VX_ID_AMD, VX_TYPE_GRAPH) + 0x07,
     /*! \brief OpenCL command queue. Use a <tt>\ref cl_command_queue</tt> parameter.*/
     VX_GRAPH_ATTRIBUTE_AMD_OPENCL_COMMAND_QUEUE         = VX_ATTRIBUTE_BASE(VX_ID_AMD, VX_TYPE_GRAPH) + 0x08,
+    /*! \brief OpenCL command queue. Use a <tt>\ref cl_command_queue</tt> parameter.*/
```

Comment:
Change the comment here - Its not OpenCL

Reply:
Resolved in the latest commit